contracts/SnappAuction.sol

```diff
@@ -126,12 +117,26 @@ contract SnappAuction is SnappBase {
         ) {
             createNewPendingBatch();
         }
-
+        bytes32 orderHash;
+        bytes memory orderData;
         for (uint i = 0; i < numOrders; i++) {
+            orderData = packedOrders.slice(26*i, 26);
```

Comment:
Please extract the other code that does this into a helper function and reuse instead of rewriting it.

Reply:
Good suggestion